src/main/java/org/terasology/projsndwv/genetics/Genome.java

```diff
@@ -96,7 +98,30 @@ public GeneticsComponent next() {
         };
     }
 
+    private GeneticsComponent mutate(GeneticsComponent component, List<Mutation> mutations) {
+        List<Mutation> mutationsCopy = new ArrayList<>(mutations);
+        Collections.shuffle(mutationsCopy);
+
+        for (Mutation mutation : mutationsCopy) {
+            if (random.nextFloat() < mutation.chance) {
+                return mutation.component;
+            }
+        }
+
+        return component;
+    }
+
     // TODO: GeneticsComponents should probably consist of two separated genomes. This will allow for non-binary genetics and less data in these overrides later on.
+
+    /**
+     * Registers an override mutation to be factored into combinations performed with this {@code Genome}. An override mutation
```

Comment:
Example of heavy technical detail here: "An override mutation mutates the entire genome of a parent if a given pair of alleles is expressed by the parents at a given locus" - wat 😁 That _sounds_ like perfectly valid English but it might do well with a simple example of what it can lead to in gameplay terms.